driver-scala/src/main/scala/org/mongodb/scala/gridfs/GridFSDownloadObservable.scala

```diff
@@ -35,7 +35,7 @@ case class GridFSDownloadObservable(private val wrapped: GridFSDownloadPublisher
    *
    * @return a Publisher with a single element, the corresponding GridFSFile for the file being downloaded
    */
-  def gridFSFile(): Observable[GridFSFile] = wrapped.getGridFSFile
```

Comment:
This change is required due to the use of the experimental features compiler flag in Scala 2.11